src/astro/files/__init__.py

```diff
@@ -24,6 +24,14 @@ def get_file_list(path: str, conn_id: str, **kwargs) -> XComArg:
 
     Supported filesystem: Local, HTTP, S3, GCS
 
+    Example path pattern:
```

Comment:
@pankajastro we can move the file pattern section from load_file.rst to concept sections and update reference in both get_file_list.rst and load_file rst.

Reply:
Fixed here 9b436a9190564e4067c3f643859d5834c83223fb